content/en/logs/guide/logs-not-showing-expected-timestamp.md

```diff
@@ -15,66 +15,115 @@ further_reading:
-        Imagine that the actual timestamp of the log is contained in the attribute mytimestamp.
-        {{< img src="logs/guide/log_timestamp_6.png" alt="log with mytimestamp attribute" style="width:75%;">}}
-        To make sure this attribute value is taken to override the log date, you must add it in the list of Date attributes.
-        The date remapper looks for each of the reserved attributes in the order in which they are configured in the reserved attribute mapping. To ensure the date comes from the `mytimestamp` attribute, place it first in the list.
-        **Note**: Any modification on the Pipeline only impacts new logs as all the processing is done at ingestion.
-        There are specific date formats to respect for the remapping to work. The recognized date formats are: [ISO8601][7], [UNIX (the milliseconds EPOCH format)][8] and [RFC3164][9].
-        If the format is different from one of the above (so if your logs still do not have the right timestamp), there is a solution.
-
-    3.2 **Custom Date format**.
-        If the format is not supported by the remapper by default, parse this format and convert it to a supported format. To do this use a [parser Processor][4] that applies only on the attribute.
-        If you do not have a Pipeline filtered on those logs yet, create a new one and add a Processor.
-        **Note**: Set this Processor only to apply to the custom `mytimestamp` attribute under the **advanced** settings.
-        {{< img src="logs/guide/log_timestamp_7.png" alt="Advanced settings date Processor" style="width:75%;">}}
-        Then define the right parsing rule depending on your date format. See the [parsing dates examples][2].
-        Add a Log Date Remapper and to have the correct timestamp on new logs.
-        {{< img src="logs/guide/log_timestamp_8.png" alt="Pipeline example" style="width:75%;">}}
-        {{< img src="logs/guide/log_timestamp_9.png" alt="Log post processing after previous pipeline" style="width:75%;">}}
+By default, when logs are received by the Datadog intake API, a timestamp is generated and appended as a date attribute. However, this default timestamp does not always reflect the actual timestamp that might be contained in the log itself. This guides walks you through how to override the default timestamp with the actual timestamp.
+
+{{< img src="logs/guide/log_timestamp_1.png" alt="Log panel showing the log timestamp that is different from the timestamp in the message" style="width:70%;">}}
+
+## Displayed timestamp
+
+The log timestamp is located at the top section of the log panel. Timestamps are stored in UTC and displayed in the user's local timezone. In the above screenshot, the local profile is set to `UTC+1`, therefore the time the log was received is `11:06:16.807 UTC`.
+
+To check if the reason the timestamp isn't showing the expected value due to the timezone being incorrectly set, go to your [Personal Settings][1] and check the Time zone section.
```

Comment:
```suggestion
The timestamp may not show the expected value because the timezone is incorrectly set. To check if this is the case, go to [Personal Settings > Preferences][1] and look at the **Time zone** section.
```